lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -326,6 +326,12 @@ sub engine_workit ($job, $callback) {
 
     log_debug "Job settings:\n" . format_settings(\%vars);
 
+    # start pre-job hook if any
+    if ($job_settings->{SYNC_ASSETS_HOOK}) {
+        log_debug 'Running SYNC_ASSETS_HOOK';
+        system($job_settings->{SYNC_ASSETS_HOOK});
+    }
```

Comment:
Since this is essentially just a pre-job hook as stated by your own comment it would probably make sense to call the setting `JOB_PRE_EXECUTION_HOOK` as well.

Reply:
I was considering this, but then it turned out to be needed at developer mode unpause too...